doc/build.rst

```diff
@@ -1,17 +1,39 @@
 目前在centos7.1测试通过
 
-开始编译
-""""""""""""""""""""
-git clone https://github.com/baidu/bigflow.git
+编译
+"""""""""""""""""""" 
 
-cd bigflow/build_support
+::
 
-sh build_deps.sh
+    git clone https://github.com/baidu/bigflow.git
+    cd bigflow/build_support
+    sh build_deps.sh
+    source ./environment
+    mkdir -p ../build && cd ../build && cmake ..
+    make
+    make release
+    
+测试
+"""""""""""""""""""""
 
-source ./environment
+::
 
-mkdir -p ../build && cd ../build && cmake ..
+    cd bigflow_python/python/
+    sh run-tests
+    
+    
+使用Docker构建
+""""""""""""""""""""""
 
-make
+运行`docker run -i -t -v $PWD:/root/bigflow centos:7.1.1503 /bin/bash`进入centos docker镜像
 
-make release
+在镜像中执行：::
+  
+    yum install sudo -y
```

Comment:
where is the git clone step?

If you assume that user already clones bigflow's repo, then make it clear in the comment.

Reply:
You saied it. My fault.